tests/integration/relations/pgbouncer_provider/application-charm/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: application
+description: |
+  Data platform libs application charm used in integration tests.
+summary: |
+  Data platform libs application meant to be used
+  only for testing of the libs in this repository.
+
+containers:
+  pgbouncer:
+    resource: application-image
+
+resources:
+  application-image:
+    type: oci-image
+    description: OCI image for test charm
+    upstream-source: ubuntu:latest
```

Comment:
Is this needed for something? Or just to have a better structure, like a normal k8s charm?